src/main/java/com/google/roomies/util/StringConverter.java

```diff
@@ -0,0 +1,38 @@
+package com.google.roomies;
+
+import static com.google.roomies.ListingConstants.CURRENCY_CODE;
+import static com.google.roomies.ListingConstants.DATE_FORMAT;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import javax.money.CurrencyUnit;
+import javax.money.Monetary;
+import org.javamoney.moneta.Money;
+
+public final class StringConverter {
+  private StringConverter() {}
+
+  /**
+  * Converts a string to a Date in the format "yyyy-MM-dd."
+  *
+  * @param date string date
+  * @return a Date in the format "yyyy-MM-dd."
```

Comment:
This isn't really true - you're returning a `Date` object, which by design is independent of the string representation.